librad/src/git/replication.rs

```diff
@@ -82,26 +82,26 @@ pub struct Config {
     pub fetch_limit: fetch::Limit,
 }
 
-pub enum Replication {
-    Clone {
-        urn: Urn,
-        identity: SomeIdentity,
-        fetched_peers: BTreeSet<PeerId>,
-    },
-    Fetch {
-        urn: Urn,
-        identity: SomeIdentity,
-        existing: BTreeSet<PeerId>,
-    },
+/// The success outcome of [`self::replicate`].
+pub struct ReplicateResult {
```

Comment:
It would be useful if we distinguished between `Cloned` and `Fetched`. For example, this would allow us to distinguish that a project was cloned fresh and we expect to see all the `rad/*` refs.

Reply:
How would you go about this? Just add another field `what_happened: ClonedOrFetched`?

Reply:
Ya, that would probably be the easiest. Maybe `enum ReplicationStatus` to mirror the `IdStatus` naming

Reply:
Did GH mess up your flow? :sweat_smile:

Reply:
No I don't think -- this is what ended up in master: fd7cbb2e9f2ed1011f1d044f745b343f0def96c3

Reply:
Ach so, then we're not going for this extra field? And are we following up on the `Even`/`Uneven` result for `person` in another patch?

Reply:
Yeah I’ll add that later